.eslintrc.js

```diff
@@ -15,4 +15,13 @@ module.exports = {
         },
     },
     ignorePatterns: 'dist',
+    overrides: [
```

Comment:
Discussion:

- https://expensify.slack.com/archives/C01GTK53T8Q/p1692889359641599
- https://expensify.slack.com/archives/C03TQ48KC/p1692889446895489

Also, this same change is also present in E/App: https://github.com/Expensify/App/blob/f74b3325b3622f8afdee44ce8de1a40b8895c1bc/.eslintrc.js#L166-L173